### ISSUE TITLE
Automation 2022.9 trigger label: "When X changes to Y for for {time in JSON}"

### ISSUE THREAD

**Issue by Quentame** [home-assistant/frontend#13651]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

When adding a trigger at a state for a certain period of time, it shows the JSON of the `for` attribute.

See example below

### Describe the behavior you expected

**Now having**
> When Lyon changes to rainy for for {"hours":1,"minutes":0,"seconds":0

(also note the double ___for___)

--

**Expecting**
> When Lyon changes to rainy for 1 hour

Note: I may prefer _during_ than _for_

### Steps to reproduce the issue

1. Add this YAML into a new automation
```
alias: TEST for JSON
trigger:
  - platform: state
    entity_id:
      - weather.YOUR_CITY
    to: rainy
    for:
      hours: 1
      minutes: 0
      seconds: 0
condition: []
action: []
mode: single
```
2. replace `weather.YOUR_CITY` by your weather entity
3. Click "Edit in the UI"
4. See the trigger label

### What version of Home Assistant Core has the issue?

container-2022.9

### What was the last working version of Home Assistant Core?

None

### In which browser are you experiencing the issue with?

Safari 15.6, Edge 103

### Which operating system are you using to run this browser?

macOS 10.15, Windows 10

**Comment by bdr99** [comment]
I think this has been fixed by https://github.com/home-assistant/frontend/pull/13723.

**Comment by Quentame** [comment]
Fixed in 2022.10
